assets/dev/js/editor/document/hooks/ui/create/column-is-populated.js

```diff
@@ -24,6 +24,7 @@ export class ColumnIsPopulated extends After {
 		containers.forEach( ( /* Container */ container ) => {
 			if ( 'column' === container.model.get( 'elType' ) ) {
 				container.view.changeChildContainerClasses();
+				container.view.moveResizeHandle();
```

Comment:
Create a dedicated hook for __moveResizeHandle__, move logic from method __moveResizeHandle__ to the hook.